core/common/src/main/java/alluxio/underfs/ObjectUnderFileSystem.java

```diff
@@ -557,6 +561,7 @@ public InputStream open(String path, OpenOptions options) throws IOException {
 
   @Override
   public boolean renameDirectory(String src, String dst) throws IOException {
+    @SuppressWarnings("nullness")
```

Comment:
should we just call it `@Nullable` since it might be null?

Reply:
Done